core/src/main/scala/sttp/capabilities/StreamMaxLimitExceeded.scala

```diff
@@ -0,0 +1,5 @@
+package sttp.capabilities
+
+case class StreamMaxLengthExceeded(maxBytes: Long) extends Exception {
```

Comment:
I think most (all?) custom exceptions that we define end with `Exception`. So maybe use this convention here as well?